ci/build_wheel.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION
+
+
+rapids-logger "Install CUDA Toolkit"
+yum update
+yum install -y epel-release
+yum install -y nvidia-driver-latest-dkms
+yum install -y cuda-toolkit-12-3
+
+rapids-logger "Install GCC"
+yum install -y centos-release-scl
+yum install -y devtoolset-9
+
+source scl_source enable devtoolset-9
+
+rapids-logger "Check conda environment"
+conda list
+
+rapids-logger "Build wheel"
+export SCCACHE_S3_NO_CREDENTIALS=1
```

Comment:
Why is this needed? I think we're probably missing some other piece of setup like https://github.com/rapidsai/shared-workflows/blob/9638706c8c3e43bb0b210528a814a2abb2f3a923/.github/workflows/wheels-build.yaml#L126-L130

Reply:
Without this, I get 
```
      sccache /usr/bin/c++    -o CMakeFiles/cmTC_1b1f2.dir/testCXXCompiler.cxx.o -c /__w/pynvjitlink/pynvjitlink/build/cp310-cp310-manylinux_2_17_x86_64/CMakeFiles/CMakeScratch/TryCompile-DBkCuI/testCXXCompiler.cxx
      sccache: error: Timed out waiting for server startup. Maybe the remote service is unreachable?
```
during the wheel build step, this is a workaround. Still following up on this.

Reply:
I think the follow-up task here is to try using the `aws-actions/configure-aws-credentials@v3` step that I linked above. It doesn't matter for this PR, though.